### DISCUSSION TITLE
Welcome new committer: Li Xing Peng

### DISCUSSION THREAD

**Discussion by seeflood** [mosn/layotto#781] [Announcements]
Hi Community,

On behalf of the Layotto PMC, I am pleased to announce that Li Xing Peng a.k.a @LXPWing , has been voted in as Layotto committer.
![image](https://user-images.githubusercontent.com/26001097/189262118-a5b201b3-d61d-48b8-9286-6a4839498d1d.png)

Li Xing Peng has been making great contributions to Layotto. Please join me to say congratulations to him!

@LXPWing , would you please briefly introduce yourself to the community?

Best Regards.

**Comment by LXPWing** [comment]
Hi everyone, I am Li Xingpeng. I am a graduating student who likes Cloud Native. I learned a lot about cloud native from the layotto community. Thank the community for the opportunity. Thanks again.

**Comment by zhenjunMa** [comment]
welcome~

**Comment by Xunzhuo** [comment]
Welcome!

**Comment by khotyn** [comment]
Congrats~

**Comment by wenxuwan** [comment]
Welcome!

**Comment by paojiao-fengzhua** [comment]
Welcome!

**Comment by Kelei0215** [comment]
Welcome～
